manage.py

```diff
@@ -17,11 +19,15 @@
 TEST_CMD = "py.test --boxed -n10 -k 'not functional' tests/"
 FUNCTIONAL_TEST_CMD = "./functional_test.sh"
 
+TWENTY_MINUTES = 1200           # seconds
+CATEGORY_DICT_LIFETIME = 200    # iterations
```

Comment:
Have this be based on the "TWENTY_MINUTES" above (except rename to more general time_between_scrapes)